charts/mattermost-team-edition/README.md

```diff
@@ -85,7 +91,7 @@ Parameter                             | Description
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
 ```bash
-$ helm install --name my-release \
+$ helm install my-release \
```

Comment:
@vijaynag-bs I'm sorry, but this is for helm 3 and not helm 2, can you keep this? same below

Reply:
@cpanato Sorry for the mistake. Have changed